etc/kayobe/kolla.yml

```diff
@@ -524,7 +524,7 @@ kolla_build_args:
 #kolla_enable_keystone:
 #kolla_enable_keystone_federation:
 #kolla_enable_keystone_horizon_policy_file:
-#kolla_enable_kibana:
+kolla_enable_kibana: true
```

Comment:
nit: You can leave this as the default since we enable central logging:
```
group_vars/all/kolla:kolla_enable_kibana: "{{ 'yes' if kolla_enable_central_logging | bool or kolla_enable_monasca | bool else 'no' }}"

```